Makefile

```diff
@@ -145,6 +145,13 @@ scoop: all
 	@ echo "Running minimal example with SCOOP"
 	python -m scoop -n 2 rmg.py -v testing/scoop/input.py
 
+update: all
+	@ echo "Updating RMG-py"
+	git stash || true
+	git pull git@github.com:ReactionMechanismGenerator/RMG-Py.git main || git pull https://github.com/ReactionMechanismGenerator/RMG-Py.git main || true
+	make
+	git stash pop || true
```

Comment:
Users should know that this could potentially lead to conflicts if the stashed changes change lines that were updated

Reply:
How was this comment resolved?